src/basilisp/contrib/pytest/testrunner.py

```diff
@@ -192,16 +217,21 @@ def teardown(self) -> None:
         assert self._fixture_manager is not None
         self._fixture_manager.teardown()
 
+    def _import_module(self) -> runtime.BasilispModule:
+        modname = _get_fully_qualified_module_name(self.path)
+        module = importlib.import_module(modname)
+        assert isinstance(module, runtime.BasilispModule)
+        return module
```

Comment:
`py.path.local` used to have a function which did this before, so replacing it required some annoying re-implementation of that logic to figure out the module name and import it manually.